modules/humanStructure/CMakeLists.txt

```diff
@@ -1,12 +1,20 @@
-# Copyright: (C) 2018 iCub Facility - Istituto Italiano di Tecnologia
+# Copyright: (C) 2019 iCub Facility - Istituto Italiano di Tecnologia
 # Authors: Vadim Tikhanoff
 # CopyPolicy: Released under the terms of the GNU GPL v2.0.
 
 project(humanStructure)
 
+message("YARP_LIBRARIES = ${YARP_LIBRARIES}")
+foreach(target ${YARP_LIBRARIES})
+    get_property(x TARGET ${target} PROPERTY INTERFACE_LINK_LIBRARIES)
+    message("    ${target} -> ${x}")
+endforeach()
+
 set(doc_files ${PROJECT_NAME}.xml)
 source_group("DOC Files" FILES ${doc_files})
 
+include_directories(${OpenCV_INCLUDE_DIRS})
```

Comment:
You can safely remove this line, as `target_link_libraries` takes care of that.

Reply:
got it